examples/test-client/src/main.tsx

```diff
@@ -0,0 +1,51 @@
+import { createRoot } from "react-dom/client";
+import "../styles/index.css";
+import { Feeds } from "./apis/Feeds";
+import { FileSystem } from "./apis/FileSystem";
+import { GPC } from "./apis/GPC";
+import { Identity } from "./apis/Identity";
+import { Navbar } from "./components/Navbar";
+import {
+  EmbeddedZupassProvider,
+  useEmbeddedZupass
+} from "./hooks/useEmbeddedZupass";
+
+export const ZUPASS_URL =
+  process.env.ZUPASS_URL || "https://staging-rob.zupass.org";
```

Comment:
(non-blocking) move to constants like https://github.com/proofcarryingdata/zapp-template/blob/main/src/constants.ts